tools/cli/commands.js

```diff
@@ -547,7 +548,13 @@ main.registerCommand({
   // latest release to create a package if we are inside an app)
   if (options.package) {
     var packageName = options.args[0];
-
+    if (options.prototype) {
+      Console.error(
+        `The ${Console.command('--prototype')} option is not supported for packages.`
```

Comment:
Might be better to say `is no longer supported` and add a link to some article explaining why.